README.md

```diff
@@ -220,7 +220,6 @@ Change the value of the `FRONTEND_SVC_URL` variable in `ui/src/lib/api.js` to th
 * Set up the email service with email credentials as explained above.
 * Build and publish the `client_stubs` and `money` modules to the local central as follows.
 ```bash
-cd module
```

Comment:
I believe this step is written assuming user will be at the root folder. And the user has to go inside each module `client_stubs` and `money` , build the jars and push to local repository. @xlight05 @dilanSachi Is that correct?

Reply:
yes. Added `module` as a placeholder for `client_stubs` and `money`

Reply:
I see, Then we better rename. This was a comment from a user who tried to built it and said no such folder named `module` within `money`

Reply:
Added a sentence. Pls check @mohanvive / @dilanSachi